open-craft/ansible-playbooks
# [BB-5999] fix: ocim deployment failing due to undefined variable

## Description

Deployment of instances in OCIM was failing due to the variable NGINX_PROXY_ORIGIN_REGEX used for nginx configurations was not defined.

## Supporting information

[BB-5999](https://tasks.opencraft.com/browse/BB-5999)

## Testing instructions

1. New [appserver](https://manage.opencraft.com/instance/32680/edx-appserver/22909/) is being setup with this commit version
3. Check deployment proceeds without the error
